clusters/app.ci/ci-operator-configresolver/ci-operator-configresolver.yaml

```diff
@@ -111,12 +111,43 @@ items:
                     values:
                     - ci-operator-configresolver
                 topologyKey: "kubernetes.io/hostname"
+        initContainers:
+        - name: git-sync-init
+          command:
+          - /git-sync
+          args:
+          - --repo=https://github.com/openshift/release.git
+          - --branch=master
+          - --root=/tmp/git-sync
+          - --one-time=true
+          env:
+          - name: GIT_SYNC_DEST
+            value: release
+          image: gcr.io/google_containers/git-sync:v3.1.6
+          volumeMounts:
+          - name: release
+            mountPath: /tmp/git-sync
         containers:
+        - name: git-sync
+          command:
+          - /git-sync
+          args:
+          - --repo=https://github.com/openshift/release.git
+          - --branch=master
+          - --wait=30
```

Comment:
This is in seconds, right?  Let's add a comment.

I saw outliers of up to 15s in reload times when I looked into this.  Given that this is the only control `git-sync` gives over how long the directory will remain in the file system, let's use 60s.  That's also the `kubelet`'s update frequency, so we won't have any surprises.